build/_headers

```diff
@@ -1,5 +1,6 @@
 /*
   Access-Control-Allow-Origin: *
   SameSite: None; Secure
+  Cache-Control: max-age=0,no-cache,no-store,must-revalidate
```

Comment:
```suggestion
  Cache-Control: no-store
```
Is the objective to prevent caching entirely?  In that case I think `no-store` is enough.  I'm basing this on the 'good' example on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control)